client/my-sites/promote-post-i2/components/campaign-item-details/index.tsx

```diff
@@ -254,13 +254,17 @@ export default function CampaignItemDetails( props: Props ) {
 
 			<header className="campaign-item-header">
 				<div>
-					<div className="campaign-item-breadcrumb">
-						{ ! isLoading ? (
-							<Breadcrumb items={ navigationItems as Item[] } compact={ isSmallScreen } />
-						) : (
-							<FlexibleSkeleton />
-						) }
-					</div>
+					{ ! isRunningInJetpack ? (
```

Comment:
The `isRunningInJetpack` flag is used in the Jetpack Dashboard app, not the Jetpack mobile app. The mobile app loads a webview of the WordPress.com site. 

To hide something for the Jetpack App you need to use the function `isWpMobileApp`.

Also, you can hide elements using CSS too (if no additional code is needed to run). You have this global class to do it: `.is-mobile-app-view`
Check the file `client/my-sites/promote-post-i2/style.scss`

Reply:
Gotcha. Many thanks. I'll do the change